build.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = "de.eldoria"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
Increment the version to 0.11.1 since this is only a fix and not a feature addition or leave it as is and I will increment it as needed.